app/models/solr_document.rb

```diff
@@ -182,7 +182,11 @@ def pdc_describe_record?
 
   def files
     @files ||= begin
-      data = JSON.parse(fetch("files_ss", "[]"))
+      data = if pdc_describe_record?
+               JSON.parse(fetch("pdc_describe_json_ss", "{}"))["files"] || []
```

Comment:
This is the main change we are picking up the data from the `pdc_describe_json_ss` field since it includes the entire file list, just like `files_ss` used to.